src/main/java/org/jenkinsci/test/acceptance/po/WizardCreateAdminUser.java

```diff
@@ -74,9 +78,18 @@ public void shouldCreateTheUserSuccessfully() {
 
     public void confirmURLSettings(){
         By confirmJenkinsUrl = by.css(".btn-primary.save-configure-instance");
-        waitFor(confirmJenkinsUrl,2);
+        waitFor(driver).withTimeout(Duration.ofSeconds(time.seconds(2)))
```

Comment:
Unrelated to this change, but should help on diagnosing `InstallWizardTest` failures if they happen again